Sources/SwiftGenStringsCore/LocalizedString.swift

```diff
@@ -18,7 +18,7 @@ private let formatSpecifierRegex: NSRegularExpression = {
     return try! NSRegularExpression(pattern: pattern, options:[])
 }()
 
-public class LocalizedString: CustomStringConvertible, Equatable {
+public final class LocalizedString: CustomStringConvertible, Equatable {
```

Comment:
**suggestion** can this be `struct` instead? Then we could also remove the `init` below :)

Reply:
And also the `public static func ==(lhs: LocalizedString, rhs: LocalizedString) -> Bool` method at the bottom